<!DOCTYPE HTML>
<!--
	KristinaCeres.com
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>KristinaCeres.com</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header">
						<h1><a href="index.html">Kristina Ceres</a></h1>
						<nav id="nav">
							<ul>
								<li class="special">
									<a href="#menu" class="menuToggle"><span>Menu</span></a>
									<div id="menu">
										<ul>
											<li><a href="index.html">Home</a></li>
											<!--<li><a href="Interests.html">Interests and Philosophy</a></li>-->
											<li><a href="Research.html">Research</a></li>
											<!--<li><a href="Interests.html">Other Projects</a></li>-->
										</ul>
									</div>
								</li>
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<article id="main">
						<header>
							<h2>Research Experience</h2>
							<p>PhD Epidemiology </br> minors in applied statistics and genomics</p>
						</header>
						<section class="wrapper style5">
							<div class="inner">
								<h3> Evolution of antimicrobial resistance (AMR) in the Mycobacterium tuberculosis complex </h3>
								<li> Predicted AMR profiles of M. bovis whole genome sequences
								<li> Estimated prevalence of AMR M. bovis in human, cattle and wildlife hosts
								<li> Estimated fitness effects of AMR mutations in samples from hosts with distinct selection pressures: humans treated with tuberculosis specific drugs, cattle treated with non-tuberculosis specific drugs and wildlife that have never been treated with antimicrobials. 
								<li> Conducted genome-wide epistasis analyses to identify mutations linked to resistance-causing protein-altering mutations, which may restore protein structure.
								<li> Used forward genetic simulation to show that epistatic compensatory mutations and clonal progeny skew can increase the probability of resistance evolution in the absence of the selection pressure caused by antimicrobial use.

							<hr />
								<h3>M. bovis within host evolutionary dynamics</i> </h3>
								<li> Evaluated the empirical distribution of M. bovis mutations evolved within hosts during infection for signals of adaptation
								<li> Used forward genetic simulation to find that the combined diversity reducing forces of population bottlenecks and clonal progeny skew and a rapid mutation rate could describe the empirical distribution of de novo evolved mutations during infection
								
							<hr />
								<h3> M. bovis pangenomics</h3>
								<li> de novo assembled and annotated 3000+ M. bovis genomes
								<li>Identified gene annotation errors associated with short read sequencing inflates accessory genome count and erroneously led others to claim M. bovis has an open pangenome
								<li> Found indel variation could be useful in outbreak investigation when outbreak sequences have similar SNP patterns
								<a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000839"><i class="fas fa-external-link"></i></a>
							<hr />
							<h3> Time series analysis of disease progression</h3>
								<li> Developed a continuous time hidden Markov model to describe Johne’s disease progression in cattle.
								<li> Implemented a modified Baum-Welch expectation maximization algorithm and Newton-Raphson estimation for parameter estimation
								<li> Found two distinct progression patterns among infected animals, with a minority of infected animals that do not progress to clinical Johne’s disease
								<li><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0242683"><i class="fas fa-external-link"></i></a>

							<hr />
									
							<h3> Shark Conservation Genomics, Stanhope Lab</h3>
								<li> Analyzed historical population dynamics of the great hammerhead and shortfin mako using the first complete genomes of the species
								<li> Detected genetic signatures of inbreeding and found high levels of homozygosity in the great hammerhead
								<li> Studying the molecular evolution of electroreception in elasmobranchs by analyzing signatures of convergent evolution in ion channels
								<li> Studying the recent demographic history of the great hammerhead using low coverage sequence data from geographically distinct populations 
								<li><a href="https://www.sciencedirect.com/science/article/pii/S2589004222020880"><i class="fas fa-external-link"></i></a>
	
							<hr />
							
							<h3> FDA Vet-LIRN – Cornell Microbial Genomics </h3>
								<li> Characterized pangenomic diversity and gene flow of E. coli samples from dog and human hosts
								<li> Used association mining to describe regional antibiogram patterns of E. coli samples from dog hosts
								<li> Conducted microbial pangenome wide association analysis of genes associated with antimicrobial resistance in E. coli samples from dog hosts
							<hr />
							
							<h3> Additional projects </h3>
								<li> Dated the introductions and characterize geospatial spread of M. bovis into the Dominican Republic as part of a collaborative project with the USDA National Veterinary Services Laboratories
								<li> Analyzed the diagnostic performance of a new canine brucellosis test developed by the Wagner Lab at the Animal Health Diagnostic Center in the absence of a gold standard
								<li> Convergent evolution analysis of Feline Infections Peritonitis Virus<a href="https://www.biorxiv.org/content/10.1101/2023.01.11.523607v1.abstract"><i class="fas fa-external-link"></i></a>
								<li> Conducted temporal dating and host dynamics analysis of a canine coronavirus with recent zoonotic spillover<a href="https://www.mdpi.com/1999-4915/14/5/853"><i class="fas fa-external-link"></i></a>
								<li> Aided in pangenomic analysis of Mycobacterium avium ssp. paratuberculosis <a href="https://journals.asm.org/doi/full/10.1128/AEM.02570-20"><i class="fas fa-external-link"></i></a>
							<hr />
									
							</div>
						</section>
					</article>

				<!-- Footer -->
					<footer id="footer">
						<ul class="icons">
							<li><a href="https://twitter.com/KristinaCeres" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="mailto: kc649@cornell.edu" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
							<li><a href="https://www.linkedin.com/in/kristina-ceres-75ba22113/" class="icon brands fa-linkedin"><span class="label">LinkedIn</span></a></li>
							<li><a href="https://github.com/kmceres/" class="icon brands fa-github"><span class="label">github</span></a></li>

						</ul>
						<ul class="copyright">
							<li>&copy; Kristina Ceres 2023</li>
						</ul>
					</footer>
					

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
