<!DOCTYPE HTML>
<!--
	KristinaCeres.com
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>KristinaCeres.com</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header">
						<h1><a href="index.html">Kristina Ceres</a></h1>
						<nav id="nav">
							<ul>
								<li class="special">
									<a href="#menu" class="menuToggle"><span>Menu</span></a>
									<div id="menu">
										<ul>
											<li><a href="index.html">Home</a></li>
											<li><a href="Interests.html">Interests and Philosophy</a></li>
											<li><a href="Research.html">Research</a></li>
											<li><a href="Interests.html">Other Projects</a></li>
										</ul>
									</div>
								</li>
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<article id="main">
						<header>
							<h2>PhD Research</h2>
							<p>Concentration in population medicine and epidemiology,</br> minors in applied statistics and genomics</p>
						</header>
						<section class="wrapper style5">
							<div class="inner">
								<h3> HMM for infectious disease progression </h3>
								<p>Infectious disease management relies on accurate characterization of disease progression so that transmission can be prevented. Slowly progressing infectious diseases can be difficult to characterize because of a latency period between the time an individual is infected and when they show clinical signs of disease. The introduction of Mycobacterium avium ssp. paratuberculosis (MAP), the cause of Johne’s disease, onto a dairy farm could be undetected by farmers for years before any animal shows clinical signs of disease. In this time period infected animals may shed thousands of colony forming units. Parameterizing trajectories through disease states from infection to clinical disease can help farmers to develop control programs based on targeting individual disease state, potentially reducing both transmission and production losses due to disease. We suspect that there are two distinct progression pathways; one where animals progress to a high-shedding disease state, and another where animals maintain a low-level of shedding without clinical disease. We fit continuous-time hidden Markov models to multi-year longitudinal fecal sampling data from three US dairy farms, and estimated model parameters using a modified Baum-Welch expectation maximization algorithm. Using posterior decoding, we observed two distinct shedding patterns: cows that had observations associated with a high-shedding disease state, and cows that did not. This model framework can be employed prospectively to determine which cows are likely to progress to clinical disease and may be applied to characterize disease progression of other slowly progressing infectious diseases. </p>
							<hr />
								<h3>Landscape pangenomics of <i> Mycobacterium bovis </i> </h3>
								<p>The purpose of this project is to determine how the M. bovis pangenome has evolved over space and time. This involves determining patterns 
								   of evolution in the core genome (genes shared by all members of the species) and the accessory genome (genes not present in all members), 
								   and to determine if these partitions of the pangenome evolve clonally. M. bovis is thought to be a strictly clonal pathogen, with no known 
								   mechanisms for ongoing horizontal gene transfer. Therefore, my overarching hypothesis is that the accessory genome evolves through serial 
								   gene deletions, and that these deletions, along with purifying selection and serial bottlenecks created by transmission, causes strong population 
								   structure. This strong population structure will lead to genetic signatures that are specific to and diagnostic of geographic location. </p>
							<hr />
								<h3>Deep learning for outbreak sequence analysis</h3>
								<p>Whole genome sequencing analysis of <i>M. bovis </i>outbreak samples is used extensively by the USDA to trace outbreaks to their source; however, 
								   there is no existing method to reliably estimate how long a herd has been infected for at the time of detection. We believe that the 
								   answer to this question lies in <i>M. bovis</i> outbreak sequence data, but the right method for detecting this signal doesn’t yet exist.
								   We're using deep learning to develop predicitive models from outbreak sequence data. More to come soon!</p>
						
							</div>
						</section>
					</article>

				<!-- Footer -->
					<footer id="footer">
						<ul class="icons">
							<li><a href="https://twitter.com/KristinaCeres" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="mailto: kc649@cornell.edu" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
						</ul>
						<ul class="copyright">
							<li>&copy; Kristina Ceres 2021</li>
						</ul>
					</footer>
					

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
